list.txt

```diff
@@ -20,13 +22,17 @@ avanceservices.us
 axelon.com
 axiomglobal.com
 bcforward.com
+BravoTech.com
 bytesys.net
 canbayinc.com
 cbsinfosys.com
 codeforce.com
 compnova.com
 compugain.com
+concept-inc.com
 consultnet.com
+cybercoders.com
+mail.cybercoders.com
```

Comment:
Does your filter not block the entire domain? I know the gmail filter does which makes this redundant.